### ISSUE TITLE
identi.ca RT not displayed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#130]
```
Hi,

identi.ca RT aren't displayed correctly. Actually, it seems only RT sent
via the identi.ca website are affected by this issue as I can see some RT's
correctly displayed in my timeline (those beginning with ♺ @.. and RT @..
are OK).

I attach a screenshot of how RT look like.

Also note that I can see the full RT text in the notification, hence it
must only be a display issue.

Also note I do not see these RT in the answers sent to me.

Cheers,
Julien

```

Original issue reported on code.google.com by `Valro...@gmail.com` on 13 Mar 2010 at 9:11

Attachments:
- [Capture-pino.png](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-130/comment-0/Capture-pino.png)

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 13 Mar 2010 at 9:12
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
RT not displayed correctly are those made via the RT button on the identi.ca 
website.

Cheers,
Julien
```

Original comment by `Valro...@gmail.com` on 14 Mar 2010 at 8:07

**Comment by GoogleCodeExporter** [comment]
```
Closed by 752cfc810849 revision.
```

Original comment by `tro...@gmail.com` on 25 Mar 2010 at 2:06
- Changed state: **Fixed**
